src/views/splash/splash.jsx

```diff
@@ -1,42 +1,38 @@
 var connect = require('react-redux').connect;
 var injectIntl = require('react-intl').injectIntl;
-var omit = require('lodash.omit');
 var React = require('react');
 var render = require('../../lib/render.jsx');
 
-var permissionsActions = require('../../redux/permissions.js');
-var sessionActions = require('../../redux/session.js');
 var shuffle = require('../../lib/shuffle.js').shuffle;
+var omit = require('lodash.omit');
 
-var Api = require('../../mixins/api.jsx');
+var permissions = require('../../redux/permissions.js');
+var session = require('../../redux/session.js');
+var rows = require('../../redux/splash/splash-rows.js');
+var activity = require('../../redux/splash/activity.js');
+var news = require('../../redux/splash/news.js');
+var projectCount = require('../../redux/splash/project-count.js');
+var homepageCache = require('../../redux/splash/homepage-cache.js');
+var templateCue = require('../../redux/splash/template-cue.js');
+
+var GlobalRows = require('./global.jsx');
+var CustomRows = require('./custom.jsx');
+var ShuffledRows = require('./shuffled.jsx');
+var SplashAdmin = require('./admin.jsx');
+var TeacherBanner = require('./teacher-banner.jsx');
 
-var Activity = require('../../components/activity/activity.jsx');
-var AdminPanel = require('../../components/adminpanel/adminpanel.jsx');
+var Modal = require('../../components/modal/modal.jsx');
 var DropdownBanner = require('../../components/dropdown-banner/banner.jsx');
-var Box = require('../../components/box/box.jsx');
-var Button = require('../../components/forms/button.jsx');
-var Carousel = require('../../components/carousel/carousel.jsx');
+var Activity = require('./activity.jsx');
 var Intro = require('../../components/intro/intro.jsx');
-var Modal = require('../../components/modal/modal.jsx');
 var News = require('../../components/news/news.jsx');
 var Page = require('../../components/page/www/page.jsx');
-var TeacherBanner = require('../../components/teacher-banner/teacher-banner.jsx');
 var Welcome = require('../../components/welcome/welcome.jsx');
 
-require('./splash.scss');
-
-var Splash = injectIntl(React.createClass({
-    type: 'Splash',
-    mixins: [
-        Api
-    ],
+var View = injectIntl(React.createClass({
+    type: 'View',
     getInitialState: function () {
         return {
-            projectCount: 14000000, // gets the shared project count
-            activity: [], // recent social actions taken by users someone is following
-            news: [], // gets news posts from the scratch Tumblr
-            featuredCustom: {}, // custom homepage rows, such as "Projects by Scratchers I'm Following"
-            featuredGlobal: {}, // global homepage rows, such as "Featured Projects"
             showEmailConfirmationModal: false, // flag that determines whether to show banner to request email conf.
```

Comment:
Looks like this one is too... but that looks like an existing mistake. The only local state in the view I see now is `emailConfirmationModalOpen` — I think that's what this key should be.